src/status_im/transport/message/transit.cljs

```diff
@@ -44,14 +44,36 @@
   (rep [this {:keys [name profile-image address fcm-token]}]
     #js [name profile-image address fcm-token]))
 
+;; It's necessary to support old clients understanding only older, verbose command content (`release/0.9.25` and older)
+(defn- new->legacy-command-data [{:keys [command-path params] :as content}]
+  (get {["send" #{:personal-chats}]    [{:command-ref ["transactor" :command 83 "send"]
+                                         :command "send"
+                                         :bot "transactor"
+                                         :command-scope-bitmask 83}
+                                        constants/content-type-command]
+        ["request" #{:personal-chats}] [{:command-ref ["transactor" :command 83 "request"]
+                                         :request-command-ref ["transactor" :command 83 "send"]
+                                         :command "request"
+                                         :request-command "send"
+                                         :bot "transactor"
+                                         :command-scope-bitmask 83
+                                         :prefill [(get params :asset)
+                                                   (get params :amount)]}
+                                        constants/content-type-command-request]}
+       command-path))
+
 (deftype MessageHandler []
   Object
   (tag [this v] "c4")
   (rep [this {:keys [content content-type message-type clock-value timestamp]}]
-    (if (= content-type constants/text-content-type)
-      ;; append new content add the end, still pass content the old way at the old index
+    (condp = content-type
+      constants/text-content-type ;; append new content add the end, still pass content the old way at the old index
```

Comment:
please make a choice here either text-content-type or content-type-text to match content-type-command or command-content-type. Anyway personally I would prefer namespaced keywords but that would be for another PR

Reply:
Ok, I will update it.